service-rpc/src/main/thrift/rpc.thrift

```diff
@@ -303,6 +303,8 @@ service TSIService {
 
 	TSExecuteBatchStatementResp testInsertTablet(1:TSInsertTabletReq req);
 
+  TSExecuteBatchStatementResp testInsertTablets(1:TSInsertTabletsReq req);
```

Comment:
Hi~  You should also update the rpc-changlist.md

Reply:
fixed